test/functional/Java11andUp/src/org/openj9/test/varhandle/TestVarHandleInfo.java

```diff
@@ -0,0 +1,115 @@
+/*******************************************************************************
+ * Copyright (c) 2019, 2019 IBM Corp. and others
+ *
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/
+ * or the Apache License, Version 2.0 which accompanies this distribution and
+ * is available at https://www.apache.org/licenses/LICENSE-2.0.
+ *
+ * This Source Code may also be made available under the following
+ * Secondary Licenses when the conditions for such availability set
+ * forth in the Eclipse Public License, v. 2.0 are satisfied: GNU
+ * General Public License, version 2 with the GNU Classpath
+ * Exception [1] and GNU General Public License, version 2 with the
+ * OpenJDK Assembly Exception [2].
+ *
+ * [1] https://www.gnu.org/software/classpath/license.html
+ * [2] http://openjdk.java.net/legal/assembly-exception.html
+ *
+ * SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0 WITH Classpath-exception-2.0 OR LicenseRef-GPL-2.0 WITH Assembly-exception
+ *******************************************************************************/
+//package java.lang.invoke;
+package org.openj9.test.varhandle;
+
+import static org.testng.AssertJUnit.assertEquals;
+import static org.testng.AssertJUnit.assertTrue;
+
+import java.lang.invoke.MethodHandle;
+import java.lang.invoke.MethodHandleInfo;
+import java.lang.invoke.MethodHandles;
+import java.lang.invoke.MethodHandles.Lookup;
+import java.lang.reflect.Modifier;
+import java.lang.invoke.MethodType;
+import java.lang.invoke.VarHandle;
+import java.util.Arrays;
+import java.util.HashMap;
+import java.util.List;
+import java.util.stream.Collectors;
+
+import org.testng.annotations.BeforeSuite;
+import org.testng.annotations.Test;
+import org.testng.log4testng.Logger;
+
+@Test(groups = { "level.extended" })
+public class TestVarHandleInfo {
+
+	protected static Logger logger = Logger.getLogger(TestVarHandleInfo.class);
+	private static final String TEST_VAR_HANDLE_INFO = "Lorg/openj9/test/varhandle/TestVarHandleInfo;"; //$NON-NLS-1$
+	private static final String INTEGER = "Ljava/lang/Integer;"; //$NON-NLS-1$
+	private static final int PUBLIC_FINAL_NATIVE = Modifier.PUBLIC + Modifier.FINAL + Modifier.NATIVE;
+	private static final int expectedModifiers = PUBLIC_FINAL_NATIVE;
+	private static final String EXPECTED_MODIFIERS_STRING = Modifier.toString(expectedModifiers);
+	private static final int expectedReferenceKind = MethodHandleInfo.REF_invokeVirtual;
+	
+	Lookup mylookup;
```

Comment:
This can be initialized here once as it won't change - no need to delay to the BeforeSuite method